plugins/es-analysis-common/src/test/java/io/crate/analysis/common/FulltextITest.java

```diff
@@ -53,7 +53,7 @@ public void testMatchOptions() throws Exception {
         execute("refresh table locations");
 
         execute("select name, _score from locations where match((kind, name_description_ft), 'galaxy') " +
-                "using best_fields with (analyzer='english') order by _score desc");
+                "using best_fields order by _score desc");
```

Comment:
`kind` was indexed with `keyword` analyzer and `name_description_ft` with `english` so `with (analyzer='english')` was actually incorrect.